README.md

```diff
@@ -1,3 +1,51 @@
+# MCAN
+
+The M\_CAN is a CAN IP module that can be realized as a standalone
+device, as part of an ASIC or on an FPGA. It performs communication
+according to ISO11898-1:2015. It supports Classical CAN and CAN FD
+(CAN with Flexible Data-rate). Additional transceiver hardware is
+required for connection to the CAN physical layer. The message
+storage is intended to be a single or dual-ported Message RAM
+outside of the module. It is connected to the M\_CAN via the Generic
+Master Interface. Depending on the chosen integration, multiple M\_CAN
+controllers may share the same Message RAM. The Host CPU is connected
+via the 32-bit Generic Interface.[^1]
+
+[^1]: [Bosch M\_CAN](https://www.bosch-semiconductors.com/ip-modules/can-ip-modules/m-can/)
+
+## Repository content
+
+This repository provides two crates:
+
+### mcan
+
+It contains platform-agnostic HAL for MCAN. It supports
```

Comment:
Nit: Many sentences start with 'it', so this might make it less repetetive:

```suggestion
It contains a platform-agnostic HAL for MCAN, with support for
```

I always thought that `mcan` *was* the abstraction layer, but when I think about it, it makes sense that a crate would contain a HAL.

Reply:
Done